docs/en/enterprise-edition/policy-reference/aws-policies/aws-iam-policies/bc-aws-355.adoc

```diff
@@ -38,5 +36,30 @@ For more info, visit https://cloudsplaining.readthedocs.io/en/latest/glossary/re
 
 Firstly, check your AWS IAM policy and identify if there are any resources where action is allowed to "*". This kind of policy allows full access to all resources which can potentially lead to security risks like unauthorized access to data, service disruption etc. 
```

Comment:
Review your IaC IAM policies to identify if any resources are set to "*", which grants full access to all resources of that type. For each identified statement, replace "*" with the specific Amazon Resource Names (ARNs) corresponding to the resources users or roles actually need access to.
The following Terraform code example describes how to define specific resources in the `Resource` field of the `Statement` block, replacing "*".